Client/src/Components/NavBar/index.css

```diff
@@ -1 +1,29 @@
-/* NavBar Component Styles*/
\ No newline at end of file
+/* NavBar Component Styles*/
+.icon-button-toggle {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.icon-button-avatar {
+  width: 25px;
+  height: 25px;
+  margin-right: var(--env-var-spacing-1);
+}
+
+.icon-button-toggle-title {
+  font-size: var(--env-var-font-size-medium);
+  color: var(--env-var-color-5);
+  margin-right: var(--env-var-spacing-2);
+}
+
+.icon-button-toggle-pic {
+  width: 10px;
+  height: 5px;
+}
+
+#icon-button {
+  color: transparent;
+  -webkit-transition: none;
+  transition: none;
+}
```

Comment:
Hey! If you want to get rid of the black outline that pops up when you click on the user dropdown menu you can use this here:
```CSS
outline: none; 
box-shadow: none;

Reply:
You can also create a new CSS class like this to get rid of the greyish hue that pops up when you hover over the user dropdown. 
```CSS
#icon-button:hover {
  background-color: transparent; 
}

Reply:
The first part worked, but the second part did not make a different and still effects on drop down menu do not disappear. By the way I think it's pretty much what we wanted on the UI.
Next, I'll be adding routing and navigation on this pr.

Reply:
I just added navigation for side menu using useNavigate and now pages render one the right side